spec/features/airline_pagination_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+feature "paginating airlines" do
+  scenario "user visits index and does not see 26th airline" do
+    25.times do
```

Comment:
use create_list instead of passing in a block.  So `FactoryGirl.create_list(:airline, name: Faker::Company.name, 25)`